src/Route.php

```diff
@@ -17,25 +17,60 @@ class Route
     /** @var mixed */
     public $handler;
 
+    public bool $isStatic = false;
+
     /**
      * @param mixed   $handler
      * @param mixed[] $variables
      */
-    public function __construct(string $httpMethod, $handler, string $regex, array $variables)
+    public function __construct(string $httpMethod, $handler, string $regex, array $variables, bool $isStatic = false)
```

Comment:
Using objects for static matching doesn't make much sense to me since we only need the handler for static routes.

Reply:
I would have to check the code, it makes sense in my fork of the lib its just not used in this PR anymore I think.